app/javascript/react/assets/styles/Typography/Typography.style.tsx

```diff
@@ -0,0 +1,107 @@
+import styled, { css } from "styled-components";
+
+const H1 = styled.h1`
+  font-family: "Roboto";
+  font-weight: 600;
+  font-size: 28px;
+  font-style: normal;
+  line-height: 1.2;
+  color: #3e4449;
```

Comment:
Please add this color to variables, also we can set the font color globally, as most often that would be true, can be then removed from all the elements.